src/components/carsoulSection/carsoureItem.js

```diff
@@ -0,0 +1,162 @@
+      authorName: 'Alice Mayer',
+      bookName: 'The Shadows: A Novel',
+      type: 'formate:Hardback',
+      price: '$31.5',
+      rate: 3,
+      views: '22,333',
+      id: 1,
+    },
+    {
+      cover:
+        'http://www.snugglyoranges.com/wp-content/uploads/2014/09/Unspoken.jpg',
+      authorName: 'Alice Mayer',
+      bookName: 'The Shadows: A Novel',
+      type: 'formate:Hardback',
+      price: '$31.5',
+      rate: 3,
+      views: '22k',
+      id: 2,
+    },
+    {
+      cover:
+        'https://s-media-cache-ak0.pinimg.com/736x/78/c5/f0/78c5f0de85565a0fd4a911f2681f56e3--book-cover-art-cover-books.jpg',
+      authorName: 'Alice Mayer',
+      bookName: 'The Shadows: A Novel',
+      type: 'formate:Hardback',
+      price: '$31.5',
+      rate: 3,
+      views: '22k',
+      id: 3,
+    },
+    {
+      cover:
+        'https://th.bing.com/th/id/OIP.Zf2SkeIiSI4ymvT2cc4u2AHaKd?pid=Api&rs=1',
+      authorName: 'Alice Mayer',
+      bookName: 'The Shadows: A Novel',
+      type: 'formate:Hardback',
+      price: '$31.5',
+      rate: 3,
+      views: '22k',
+      id: 4,
+    },
+    {
+      cover:
+        'https://s-media-cache-ak0.pinimg.com/736x/78/c5/f0/78c5f0de85565a0fd4a911f2681f56e3--book-cover-art-cover-books.jpg',
+      authorName: 'Alice Mayer',
+      bookName: 'The Shadows: A Novel',
+      type: 'formate:Hardback',
+      price: '$31.5',
+      rate: 3,
+      views: '22k',
+      id: 5,
+    },
+    {
+      cover:
+        'https://s-media-cache-ak0.pinimg.com/736x/78/c5/f0/78c5f0de85565a0fd4a911f2681f56e3--book-cover-art-cover-books.jpg',
+      authorName: 'Alice Mayer',
+      bookName: 'The Shadows: A Novel',
+      type: 'formate:Hardback',
+      price: '$31.5',
+      rate: 3,
+      views: '22k',
+      id: 6,
+    },
+  ];
+
+  let createCards = () => {
+    let specialBooks = [];
+    specialBooks = booksArray.slice(0, 4);
+
+    cards = specialBooks.map((book, index) => {
+      return (
+        <MDBCarouselItem
+          itemId={index}
+          key={book.id}
+          className="mt-md-4 mb-md-4 pl-lg-3"
+        >
+          <MDBCard className="w-30 shadow-none" wide>
+            <MDBContainer className="w-full">
+              <MDBRow className="p-0">
+                <MDBCol size="12" md="4" lg="5" className="p-0">
+                  <img
+                    className="imageCardCarsouel"
+                    src={`${book.cover}`}
+                    alt="cover book"
+                  />
+                </MDBCol>
+
+                <MDBCol md="1" lg="1"></MDBCol>
+
+                <MDBCol md="7" lg="6" size="12" className="my-auto">
+                  <MDBCardBody className="p-lg-0 text-center text-md-left text-lg-left mx-auto">
+                    <div>
+                      <h1 className="fontHeader1 mb-0 font-weight-normal  text-2xl">
+                        {' '}
+                        {`${book.authorName}`}{' '}
```

Comment:
remove this random white spaces from here and every where in your code
```suggestion
                        {`${book.authorName}`}
```